lib/screens/v2/departure/builder.ex

```diff
@@ -58,6 +59,12 @@ defmodule Screens.V2.Departure.Builder do
 
   defp vehicle_already_departed?(_), do: false
 
+  defp prediction_has_no_departure_time?(%Schedule{}), do: false
```

Comment:
We should also do this for scheduled departures, I think (filtering them out if they have no departure time). We don't want to display an "arrival-only" stop that won't be accepting passengers, even if it's not a live prediction.